src/api/xdsl/meeting/xdsl-meeting.v6.service.js

```diff
@@ -0,0 +1,44 @@
+angular.module('ovh-api-services').service('OvhApiXdslMeetingV6', ($resource, OvhApiXdslMeeting, Poller) => {
+  const interceptor = {
+    response(response) {
+      OvhApiXdslMeeting.resetCache();
+      return response.resource;
+    },
+  };
+
+  const meeting = $resource('/xdsl/:serviceName/orderMeeting', {
```

Comment:
Regarding the `url`, both `orderMeeting` and `searchOrderMeetings` actions must be part of the following service `OvhApiXdslV6`.

Is that make sense for you?

Reply:
If you want, it's the same for me.

Reply:
This is how the project is architecture.
You can refer to the following documentation here: https://github.com/ovh-ux/ovh-api-services#structure

Thank you!